PhysicsTools/PatAlgos/plugins/PATMuonSlimmer.cc

```diff
@@ -46,6 +47,12 @@ pat::PATMuonSlimmer::PATMuonSlimmer(const edm::ParameterSet & iConfig) :
     src_(consumes<pat::MuonCollection>(iConfig.getParameter<edm::InputTag>("src"))),
     linkToPackedPF_(iConfig.getParameter<bool>("linkToPackedPFCandidates")),
     saveTeVMuons_(iConfig.getParameter<std::string>("saveTeVMuons")),
+    dropDirectionalIso_(iConfig.getParameter<std::string>("dropDirectionalIso")),
+    dropPfP4_(iConfig.getParameter<std::string>("dropPfP4")),
+    slimCaloVars_(iConfig.getParameter<std::string>("slimCaloVars")),
+    slimKinkVars_(iConfig.getParameter<std::string>("slimKinkVars")),
+    slimCaloMETCorr_(iConfig.getParameter<std::string>("slimCaloMETCorr")),
+    slimMatches_(iConfig.getParameter<std::string>("slimMatches")),
```

Comment:
Please edit the PR title and description to be clear about additional (configurable) packing of miniAOD pat leptons.